### ISSUE TITLE
[Hunter] [Spell] Traps triggered by non-faction non-hostile NPCs

### ISSUE THREAD

**Issue by mhdemn** [Atlantiss/NetherwingBugtracker#2465]
TLDR: factionless and non hostile NPCs triggering hunter traps, differs from #2200 in the faction part, if this is caused by temporary PVE mode close the issue right away and apologies for the inconvenience <3

**Description**: Pretty much what title says. This might be related to issue #2200 but that one talks about enemy faction NPCs pacified 'cause of Shattrath. In this case though: [Kylene](https://tbc-twinhead.twinstar.cz/?npc=19186) is in Shattrath but she doesn't belong to any faction, is friendly to both; [Ancient Orc Ancestors](https://tbc-twinhead.twinstar.cz/?npc=18688) are in Nagrand, are neutral to both factions and belong to none. 

**Current behaviour**: These two examples and most likely some other NPCs trigger hunter traps. [Kylene gets affected by it](https://www.youtube.com/watch?v=WfMhfy6A4lQ&feature=youtu.be) but heals up quickly thru the damage since she doesn't engage in combat. [Ancestors trigger it](https://www.youtube.com/watch?v=SEDIzNATR_s&feature=youtu.be) but don't get the debuff or any damage.

**Expected behaviour**: Not hostile = ignores your traps.

**Server Revision**: 2261

**Comment by mhdemn** [comment]
you did mention only pvp-flagable npcs, so I understood there was a difference, specially 'cause the pics you added show only this kind of npcs getting affected.
Anyhow, now it's reported so it covers issues both inside and outside Shattrath with both faction and factionless npcs.
If devs take a look at it and identify it as the same thing they'll close the issue, no biggie

**Comment by Tenoutoften** [comment]
Traps also trigger on hostile NPCs not yet activated, for example in old hillsbrad before the NPCs start fighting, but already spawned, a trap placed beneath then it will explode, dealing zero damage.

**Comment by Cosmic-Lattee** [comment]
It also seems to happen when critters move over you traps well. I have had several freeze traps break because of critters happening to be to close to them.

**Comment by The14me** [comment]
rev 3451.Still an issue.

**Comment by Wolffenstein** [comment]
Fixed in rev 3459.
